cardinal/evm/server.go

```diff
@@ -21,33 +20,37 @@ var (
 	_ routerv1grpc.MsgServer = &srv{}
 )
 
-// ITransactionTypes is a map that maps transaction type ID's to transaction types.
+// ITransactionTypes maps transaction type ID's to transaction types.
 type ITransactionTypes map[transaction.TypeID]transaction.ITransaction
 
-var _ TxHandler = &ecs.World{}
-
-// TxHandler is a type that gives access to transaction data in the ecs.World, as well as access to queue transactions.
-type TxHandler interface {
-	AddTransaction(transaction.TypeID, any, *sign.SignedPayload) uint64
-	ListTransactions() ([]transaction.ITransaction, error)
-}
+// IReadTypes maps read resource names to the underlying IRead
+type IReadTypes map[string]ecs.IRead
 
 type srv struct {
```

Comment:
not to be a troll, but feels like someone reading this code would be confused by what this is for cuz of the name? feel free to resolve/ignore if you think it doesn't matter much or there isn't a better name

Reply:
glad you called this out, cause these types shouldn't be exported either, since they're only used within this package. 

I've renamed them to describe what they're actually doing, i.e. mapping some identifier to some type - txById and readByName

Reply:
re: `srv` updated to be `msgServerImpl`